web/src/components/newFeature.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Img from 'gatsby-image';
+
+import styled from 'styled-components';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
is this necessary? I thought I had this fixed in the eslint config. If it's still throwing an error after you remove let me know.